esapi/esapi/Esys_PolicyCommandCode.c

```diff
@@ -148,11 +151,13 @@ Esys_PolicyCommandCode_async(
     r = check_session_feasability(shandle1, shandle2, shandle3, 0);
     return_if_error(r, "Check session usage");
 
-    store_input_parameters(esysContext,
-                policySession,
+    store_input_parameters(esysContext, policySession,
                 code);
```

Comment:
Align 'code' to the opening bracket or move it to the line above. It should fit.

Reply:
Fixed...